src/crypto/node/crypto.ts

```diff
@@ -14,32 +14,100 @@
 
 import * as crypto from 'crypto';
 import {Crypto} from '../crypto';
+import * as base64js from 'base64-js';
+
+interface WebCryptoInterface {
+  subtle: {
+    digest: Function;
+    importKey: Function;
+    verify: Function;
+    sign: Function;
+  };
+  getRandomValues: Function;
+}
+
+const webcrypto = (crypto as any).webcrypto as WebCryptoInterface;
 
 export class NodeCrypto implements Crypto {
   async sha256DigestBase64(str: string): Promise<string> {
-    return crypto.createHash('sha256').update(str).digest('base64');
+    const inputBuffer = new TextEncoder().encode(str);
+
+    // Result is ArrayBuffer as well.
+    const outputBuffer = await webcrypto.subtle.digest('SHA-256', inputBuffer);
+
+    return base64js.fromByteArray(new Uint8Array(outputBuffer));
   }
 
   randomBytesBase64(count: number): string {
-    return crypto.randomBytes(count).toString('base64');
+    const array = new Uint8Array(count);
+    webcrypto.getRandomValues(array);
+    return base64js.fromByteArray(array);
```

Comment:
Since we're in Node context, we still can use `Buffer.from` here, right?

Reply:
In playing I could seem to, seems like this might be a good improvement.